source/concurrent/SynchronizedQueue.ooc

```diff
@@ -58,7 +58,7 @@ BlockedQueue: class <T> extends SynchronizedQueue<T> {
 		this _populated broadcast()
 	}
 	wait: func (isOk := null as Bool*) -> T {
-		result: T
+		result: T = null
```

Comment:
`result: T = null` and `result := null as T` both work, right? We should decide on one going forward and I mostly like the one you wrote here.